index.html

```diff
@@ -31,5 +31,142 @@ <h2>Let's Connect</h2>
                 </ul>
             </div>
         </main>
+        <main id="works">
```

Comment:
- [ ] Please 🙏🏽 use the `section` tag instead of the `main` tag for semantic reasons for better practices of HTML & CSS.